tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -51,6 +51,13 @@ readonly dax_header_sz=2
 readonly dax_alignment=2
 
 # The list of systemd units and files that are not needed in Kata Containers
+readonly -a udev_systemd_units=(
```

Comment:
Could you explain me why those units are not need only and only if `DM_VERITY=no`? Sorry, didn' t get that part....

Reply:
When installing dependencies with dm-verity, we need to install udev, and udev need to be set as a startup process, so the related files of udev need to be preserved.